src/frontend/screens/MapScreen/ObsevationMapLayer.tsx

```diff
@@ -15,7 +15,7 @@ const layerStyles = {
 };
 
 export const ObservationMapLayer = () => {
-  const observations = useAllObservations();
+  const {data: observations} = useObservations();
```

Comment:
Right now the user only see the observations on the home screen map, but I have asked Sabella if they should also see the tracks.

Reply:
Added tracks as a map layer: https://github.com/digidem/comapeo-mobile/pull/408/commits/32245c48431a56ec7050794476a2acf647b22ba4